CHANGELOG.md

```diff
@@ -1,40 +0,0 @@
-# <a name="changelog"></a>CHANGE LOG
```

Comment:
The content in https://github.com/CivicActions/handbook/blob/master/CHANGELOG.md is the changelog from CivicActions handbook (the Google Drive one).